lib/access-context.js

```diff
@@ -250,16 +262,23 @@ Principal.prototype.equals = function(p) {
 
 /**
  * A request to access protected resources.
- * @param {String} model The model name
- * @param {String} property
+ *
+ * The method can either be called with the following signature or with a single
+ * argument: an AccessRequest instance or an object containing all the required properties.
+ *
```

Comment:
adding a comment to let know the one object param signature
(this should be the default doc, as for AccessContext)